examples/testing-dom__drag-drop/index.html

```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <title>Drag-Drop</title>
+</head>
+<body>
+<h1>Drag and Drop</h1>
+<ol>
+    <li><a href="http://localhost:7071/basketball.html">Basketball example.</a></li>
+    <li><a href="http://localhost:7071/puzzle.html">Puzzle example.</a></li>
```

Comment:
Use relative links instead of hardcoding localhost in case port changes one day

```suggestion
    <li><a href="/basketball.html">Basketball example.</a></li>
    <li><a href="/puzzle.html">Puzzle example.</a></li>
```